projects/ion/src/lib/dropdown/dropdown.component.ts

```diff
@@ -39,6 +39,14 @@ export class DropdownComponent implements AfterViewInit {
 
   iconSize = 16;
 
+  clearButtonIsVisible: boolean;
+
+  setClearButtonIsVisible() {
```

Comment:
```suggestion
  setClearButtonIsVisible(): void {
```

Reply:
i had already corrected this, i don't know why it came back, but i corrected again